apps/studio/pages/project/[ref]/reports/edge-functions.tsx

```diff
@@ -14,38 +14,43 @@ import DefaultLayout from 'components/layouts/DefaultLayout'
 import ReportsLayout from 'components/layouts/ReportsLayout/ReportsLayout'
 import { ButtonTooltip } from 'components/ui/ButtonTooltip'
 import { Button, Checkbox, DropdownMenu, DropdownMenuContent, DropdownMenuTrigger } from 'ui'
+import { useChartHoverState } from 'components/ui/Charts/useChartHoverState'
 
 import { useEdgeFunctionsQuery } from 'data/edge-functions/edge-functions-query'
-import { getEdgeFunctionReportAttributes } from 'data/reports/edgefn-charts'
+import { edgeFunctionReports } from 'data/reports/v2/edge-functions.config'
 
 import { REPORT_DATERANGE_HELPER_LABELS } from 'components/interfaces/Reports/Reports.constants'
 import UpgradePrompt from 'components/interfaces/Settings/Logs/UpgradePrompt'
 import { useReportDateRange } from 'hooks/misc/useReportDateRange'
 
 import type { NextPageWithLayout } from 'types'
+import { ReportSettings } from 'components/ui/Charts/ReportSettings'
 
-const EdgeFunctionsReport: NextPageWithLayout = () => {
+const EdgeFunctionsReportV2: NextPageWithLayout = () => {
   return (
     <ReportPadding>
       <EdgeFunctionsUsage />
     </ReportPadding>
   )
 }
 
-EdgeFunctionsReport.getLayout = (page) => (
+EdgeFunctionsReportV2.getLayout = (page) => (
   <DefaultLayout>
     <ReportsLayout title="Edge Functions">{page}</ReportsLayout>
   </DefaultLayout>
 )
 
-export type UpdateDateRange = (from: string, to: string) => void
-export default EdgeFunctionsReport
+export default EdgeFunctionsReportV2
 
 const EdgeFunctionsUsage = () => {
   const { ref } = useParams()
   const { data: functions, isLoading: isLoadingFunctions } = useEdgeFunctionsQuery({
     projectRef: ref,
   })
+
+  const chartSyncId = `edge-functions-${ref}`
+  const _ = useChartHoverState(chartSyncId)
```

Comment:
tiny nit - could just call the hook if we're not using the returned value
```suggestion
useChartHoverState(chartSyncId)
```